packages/frontend/components/global/CookiesBanner/CookiesBanner.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+import * as Styled from "./CookiesBanner.styles";
+
+export default function CookiesBanner({ onAccept, onClose }: Props) {
+  return (
+    <Styled.Banner className="l-container-max a-bg-custom20" role="alert">
+      <Styled.Content className="l-container-wide">
+        <p className="t-copy-sm">
+          Cookies are delicious. Vitae pharetra maecenas amet eu luctus
+          convallis. Feugiat pulvinar eleifend eu fames accumsan, ipsum diam.
+        </p>
+        <Styled.Buttons className="l-flex l-flex--gap">
```

Comment:
The gap here is 12px instead of 16px (applied by `l-flex--gap`) - I'd define the gap in `Styled.Buttons` for now, until we start to see a pattern develop.